netmap/types.proto

```diff
@@ -228,11 +228,31 @@ message NodeInfo {
   State state = 4 [json_name = "state"];
 }
 
+// NeoFS network configuration
+message NetworkConfig {
+  // Single configuration parameter.
+  message Parameter {
+    // Parameter key. Hex-encoded string.
+    bytes key = 1 [json_name = "key"];
+
+    // Parameter value.
+    bytes value = 2 [json_name = "value"];
+  }
+  // List of parameter values.
+  repeated Parameter parameters = 1 [json_name = "parameters"];
+}
+
 // Information about NeoFS network
 message NetworkInfo {
     // Number of the current epoch in the NeoFS network.
     uint64 current_epoch = 1 [json_name = "currentEpoch"];
 
     // Magic number of the sidechain of the NeoFS network.
     uint64 magic_number = 2 [json_name = "magicNumber"];
+
+    // MillisecondsPerBlock network parameter.
```

Comment:
I suggest to emphasize that it is side chain network parameter in the comment. Therefore `network_config` is a NeoFS network configuration.